control-plane/api-gateway/binding/route_binding.go

```diff
@@ -177,6 +181,17 @@ func (r *Binder) bindRoute(route client.Object, boundCount map[gwv1beta1.Section
 				})
 			}
 
+			if invalidFilterNames := authFilterReferencesMissingJWTProvider(httproute, r.config.Resources); len(invalidFilterNames) > 0 {
+				results = append(results, parentBindResult{
+					parent: ref,
+					results: []bindResult{
+						{
+							err: fmt.Errorf("%w: %s", errFilterInvalid, strings.Join(invalidFilterNames, ",")),
```

Comment:
It looks like this code path doesn't have any test coverage. If that's true, can you add coverage in this PR or queue up a ticket to add test coverage in the next couple of weeks? I know we're on a tight timeline

Reply:
good catch I can add some to cover this path to https://github.com/hashicorp/consul-k8s/blob/main/control-plane/api-gateway/binding/binder_test.go

Reply:
@jm96441n queued up a ticket